ee/billing/quota_limiting.py

```diff
@@ -101,6 +104,7 @@ class QuotaLimitingCaches(Enum):
     QuotaResource.LLM_EVENTS: "llm_events",
     QuotaResource.CDP_TRIGGER_EVENTS: "realtime_destinations",
     QuotaResource.ROWS_EXPORTED: "rows_exported",
+    QuotaResource.AI_CREDITS_USED: "ai_credits",
```

Comment:
This should be `posthog_ai` (see comment on line 95)

![image.png](https://app.graphite.com/user-attachments/assets/29f107e4-133d-4aba-9333-27d13f7938cc.png)

Reply:
On this actually, I'm confused by `ee/api/test/test_billing.py::TestBillingAPI::test_organization_usage_count_with_demo_project` failing – it looks like the entry for `ai_credits` is missing there `self.organization.usage`. How do you think we fix this test?

Reply:
FYI I've merged my PR, then merged master into your branch and resolved conflicts and pushed to remote